js/lykke2.js

```diff
@@ -1025,7 +1029,10 @@ module.exports = class lykke2 extends Exchange {
         //         "timestamp":1644146723620
         //     }
         //
-        const id = this.safeString (transaction, 'operationId');
+        let id = this.safeString (transaction, 'operationId');
```

Comment:
It will not be able to look up a key like `operationId` in a string, if `transaction` is a string.
We have to rewrite it like this:

```JavaScript
let id = undefined;
if (typeof transaction === 'string') {
    id = transaction;
} else {
    id = this.safeString (transaction, 'operationId');
    // ...
}
```